src/Rules/Debug/StaticChainedNoDebugInNamespaceRule.php

```diff
@@ -0,0 +1,47 @@
+<?php declare(strict_types=1);
+
+namespace Hihaho\PhpstanRules\Rules\Debug;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\StaticCall;
+use PHPStan\Analyser\Scope;
+use PHPStan\Rules\Rule;
+use PHPStan\Rules\RuleErrorBuilder;
+
+/**
+ * @implements \PHPStan\Rules\Rule<StaticCall>
+ */
+class StaticChainedNoDebugInNamespaceRule extends BaseNoDebugRule implements Rule
+{
+    protected string $message = 'No statically called debug statements should be present in the %s namespace.';
+
+    public function getNodeType(): string
+    {
+        return StaticCall::class;
+    }
+
+    public function processNode(Node $node, Scope $scope): array
```

Comment:
This method is identical to the one in `ChainedNoDebugInNamespaceRule`. Maybe it's something that can be extracted to a parent class.

Reply:
I have thought of introducing a base class, as the lines 29-43 also are pretty much the same.

I chose not to for the sake of being explicit in what the rule does. I also like that I can see upon opening of this file what the checks are and what node is being processed